modules/ROOT/pages/support-center-access.adoc

```diff
@@ -6,10 +6,17 @@ endif::[]
 
 image::support-logo.png[support]
 
-Your account currently doesn't have access to the Support Center. This may be because your account is a free account, a trial, or has not been fully set up according to your order. If you are a customer and you believe you have reached this page by mistake and should have access to the Support Center, please mailto:info@mulesoft.com[contact us].
+You are seeing this page because your account doesn't currently have access to the MuleSoft Support Center. This could be because your account is a free account, a trial, or is not fully set up according to your order.
```

Comment:
The use of "currently" could be construed as a future promise and could open us to litigation. Please remove.